### PR TITLE
tools: update note and db-type values

### PR DESCRIPTION
**Pull request by lilin90**
@WangXiangUSTC @CaitinChen PTAL
Via: https://github.com/pingcap/docs-cn/pull/987

### CODE REVIEW

**Comment by lilin90** [comment]
@WangXiangUSTC PTAL

File: tools/tidb-binlog-cluster.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ The server hardware requirements for development, testing, and the production en
 * If you want to start Drainer in the existing TiDB cluster, generally, you need to make a full backup of the cluster data, obtain `savepoint`, import the data to the target database, and then start Drainer to synchronize the incremental data from `savepoint`.
 * Drainer supports synchronizing binlogs to MySQL, TiDB, Kafka or the local files. If you need to synchronize binlogs to other destinations, you can set Drainer to synchronize the binlog to Kafka and read the data in Kafka for customization processing. See [Binlog Slave Client User Guide](../tools/binlog-slave-client.md).
 * If TiDB-Binlog is used for recovering the incremental data, you can set the downstream to `pb` to synchronize the binlog data to the local file and then use [Reparo](../tools/reparo.md) to recover the incremental data.
+* If the downstream is set to `tidb`, the `_tidb_rowid` hidden column of TiDB is used to synchronize data. It should be noted that only the Drainer data is allowed to write into the downstream, otherwise an error might occur to the data.
```
[the remark this answers is not in the document]
Comment (lilin90) [reply tools/tidb-binlog-cluster.md]: @GregoryIan @WangXiangUSTC Got it. So we remove this line or how to modify it?

[the remark this answers is not in the document]
Comment (GregoryIan) [reply tools/tidb-binlog-cluster.md]: - as the result we discussed, we wouldn't refer it in any document, both `TiDB` and `Tools`.
- `TiDB` already changes usage of `_tidb_rowid`, but new binlog doesn't support it
- they don't suffer enough testing

so remove it

[the remark this answers is not in the document]
Comment (GregoryIan) [reply tools/tidb-binlog-cluster.md]: but maybe we add it later if it's ready

**Comment by lilin90** [comment]
@GregoryIan @csuzhangxc I removed the `_tidb_rowid` line. To make it consistent, I'll also move that line in docs-cn later. But we do have `_tidb_rowid` shown in multiple documents now, like [TiDB Specific System Variables](https://github.com/pingcap/docs/blob/master/sql/tidb-specific.md#_tidb_rowid), [mydumper](https://github.com/pingcap/docs/blob/master/tools/mydumper.md#what-enhancements-does-this-contain-over-regular-mydumper), [sync-diff-inspector User Guide](https://github.com/pingcap/docs/blob/master/tools/sync-diff-inspector.md#description-of-common-configuration).

**Comment by GregoryIan** [comment]
@lilin90 yep, help to remove them all

**Pull request merged**